### PR TITLE
Seeder

### PR DESCRIPTION
**Pull request by alhamp**
Seeder files added.

### CODE REVIEW

File: src/main/java/com/launchacademy/giantleap/seeders/BrandSeeder.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    for(String brandName : brands) {
+      if(brandRepository.findByName(brandName) == null) {
+        Brand brand = new Brand();
+        brand.setName(brandName);
+        brandRepository.save(brand);
+      }
+    }
+  }
+}
+
```
Comment (prophetsfall) [inline src/main/java/com/launchacademy/giantleap/seeders/BrandSeeder.java]: Please remove all the extra lines here

File: src/main/java/com/launchacademy/giantleap/models/Brand.java [hunk 2]
```diff
@@ -34,4 +34,5 @@
 
   @OneToMany(mappedBy = "brand")
   private List<FashionItem> fashionItems = new ArrayList<FashionItem>();
+
```
Comment (prophetsfall) [inline src/main/java/com/launchacademy/giantleap/models/Brand.java]: please remove extra line

File: src/main/java/com/launchacademy/giantleap/models/FashionItemReview.java [hunk 3]
```diff
@@ -44,4 +44,5 @@
 
     @Column
     private Integer rating;
+
```
Comment (prophetsfall) [inline src/main/java/com/launchacademy/giantleap/models/FashionItemReview.java]: please remove extra line

**Comment by alhamp** [comment]
Made the changes

**Pull request merged**